ftplugin/julia.lua

```diff
@@ -0,0 +1,44 @@
+local util = require("lspconfig/util")
+-- The default config lazily evaluates the location of the julia language server from the your global julia packages.
+-- This adds a small overhead on first opening of a julia file. To avoid this overhead, replace server_path in on_new_config with
+-- a hard-coded path to the server.
+
+-- LanguageServer.jl can be installed with julia and Pkg
+-- julia -e 'using Pkg; Pkg.add("LanguageServer"); Pkg.add("SymbolServer")'
+
+-- for details see https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/julials.lua
+local julials = "using LanguageServer  \n"
+	.. "using LanguageServer.SymbolServer  \n"
```

Comment:
is there a reason that you excluded these lines:
```julia
    using Pkg;
    Pkg.instantiate()
```

Reply:
No good reason. I tested on random Julia files not part of a project, and so I didn't notice.
`using Pkg; Pkg.instantiate()` should definitely be there. Will add.

Reply:
And glad I can help out!

Reply:
So I think I was doubly wrong. We want to instantiate if a `Project.toml` exists, otherwise fallback to the global environment package. Working through a better way.